en/docs/references/pricing.md

```diff
@@ -0,0 +1,227 @@
+# Pricing
+
+## Introduction
+
+Choreo is a Digital Platform as a Service that helps you develop integration solutions faster by reducing the coding effort with a low-code development platform. You can use Choreo by paying only for your usage: nothing more, nothing less. Choreo has payment plans for all types of users. You can enjoy the free tier with limitations and then move on to the Pay-As-You-Go model if you use more than 5,000 [steps](#what-is-a-step) per month. If you plan to use Choreo significantly, you can also opt for an Enterprise plan and enjoy special discount rates.
+
+## Plans
+
+|               | Free                   |  Pay As You Go    | Enterprise     |
+|---------------|------------------------|-------------------|----------------|
+| Cost Per Step | 5,000 steps per month *|$5 per 1,000 steps <br/> <ul><li>First 5,000 steps per month are free</li></ul>| Contact Us For Discounted Pricing <br/><ul><li> Volume Discounts</li><li> Committed-use Discounts</li><li> Support Channels and Support SLA</li></ul>|
```

Comment:
Why do we use title caps in the **Enterprise** column? Let's use a colon before a list.

Suggestion:
"Contact us for discounted pricing offers including:
 - Volume discounts
 - Committed-use discounts
 - Support channels and support SLA"

Reply:
done